mongodb/trigger/eventlistener/descriptor.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
We need a README.md for this trigger

Reply:
I have pushed a README.md for the trigger